packages/tables/resources/views/columns/checkbox-column.blade.php

```diff
@@ -1,16 +1,18 @@
 <div
-    x-data="{ error: undefined }"
+    x-data="{ error: undefined, state: @js($getState()), loading: false }"
```

Comment:
Please rename all `loading` Alpine property references to `isLoading` to indicate the boolean type

Reply:
Also format x-data properties on new lines with trailing commas please